xarray/backends/zarr.py

```diff
@@ -591,129 +637,37 @@ def open_zarr(
     ----------
     http://zarr.readthedocs.io/
     """
-    if "auto_chunk" in kwargs:
-        auto_chunk = kwargs.pop("auto_chunk")
-        if auto_chunk:
-            chunks = "auto"  # maintain backwards compatibility
-        else:
-            chunks = None
 
-        warnings.warn(
-            "auto_chunk is deprecated. Use chunks='auto' instead.",
-            FutureWarning,
-            stacklevel=2,
-        )
```

Comment:
For now I would suggest keeping this compatibility code and not (yet) marking `open_zarr` as deprecated.

Reply:
Ok I can revert the open_zarr deprecation warning. Should I also not remove the "auto_chunk" compatibility here?